README.md

```diff
@@ -33,7 +39,8 @@ You can override specific settings by specifying them as normal. See <http://esl
 ## Publishing
 
 ```bash
-Update CHANGELOG
+# Make sure to only make do this in the package that has changed
```

Comment:
One thing of note here is that you probably don't want the tags that npm creates, unless you want to version the two packages as one